package.json

```diff
@@ -1,11 +1,16 @@
 {
-  "name": "visual",
-  "scripts": {
-    "postinstall": "pbiviz update 1.6.0",
+   "name": "PowerBI-visuals-clustering-kmeans-1.0.3",
+  "version": "1.0.3",
+  "author": {
+    "name": "Microsoft",
+    "email": "pbicvsupport@microsoft.com"
+},
+ "scripts": {
+    "postinstall": "pbiviz update 1.7.0",
     "pbiviz": "pbiviz",
+    "start": "pbiviz start",
     "package": "pbiviz package",
-    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src)/**/*.ts\"",
-    "start": "pbiviz start"
+    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src|test)/**/*.ts\""
   },  
   "devDependencies": {
     "tslint": "^4.4.2",
```

Comment:
Please use `"powerbi-visuals-tools": "1.7.1"` to install a proper version of pbiviz